src/libdrakvuf/libdrakvuf.h

```diff
@@ -385,6 +386,7 @@ bool drakvuf_init (drakvuf_t* drakvuf,
     const bool fast_singlestep,
     uint64_t limited_traps_ttl) NOEXCEPT;
 bool drakvuf_init_os (drakvuf_t drakvuf) NOEXCEPT;
+domid_t drakvuf_get_dom_id(drakvuf_t drakvuf) NOEXCEPT;
```

Comment:
Please move this to be around the same spot get_kernel_base is

Reply:
Done. I moved it just above `drakvuf_get_kernel_base(...)` 
(This was actually my first intuition...)

Hope, this place is good now.